301-metadata-repositories.md

```diff
@@ -0,0 +1,307 @@
+
+## Metadata repositories
+
+A _metadata repository_ is a service that hosts TUF and / or in-toto metadata about bundles and / or images. The repository is conceptually distinct from a [CNAB registry](200-CNAB-registries.md), which hosts bundles and / or images themselves.
+
+How TUF and in-toto metadata should be designed for a metadata repository depends on which purpose it serves. The repository may be _private_ for internal consumption, or _public_. Authentication to private metadata repositories are out of the scope of this document.
+
+### Example for multiple projects
+
+This subsection discusses how an organization (e.g., `example.com/example-org/*`) MAY setup a metadata repository that hosts metadata about bundles and / or images developed and / or maintained by different _projects_, groups of developers. This repository MAY be private or public.
+
+![Figure 1: The suggested metadata repository for bundles and / or images developed and / or maintained by different projects, or groups of developers](img/example-metadata-repository.png)
+
+Figure 1 illustrates this suggested metadata repository, which we discuss using a top-down, outside-in approach.
+
+#### Administrators
+
+The four-top level TUF `root`, `timestamp`, `snapshot`, and `targets` roles SHOULD be controlled by the repository _administrators_, who develop and maintain the repository's hardware and software.
+
+The `root` role distributes, revokes, and replaces the public keys for all four top-level roles. It SHOULD use _offline keys_, or signing keys kept off the Internet (e.g., on trusted hardware in cold storage). It SHOULD also use a threshold (m, n) of keys, where n is the number of administrators, and m is the number of quorum members that must agree on new `root` metadata. Its metadata SHOULD expire yearly, considering that an offline key ceremony is expensive in terms of time and resources.
+
+The `timestamp` role indicates concisely whether there is any new metadata, bundle, or image on the repository.
+
+The `snapshot` role prevents attackers from tampering with interdependencies between bundles and / or images.
+
+Since metadata for both the `timestamp` and `snapshot` roles could be updated whenever a new bundle/ image is produced at any time, they MAY each use a threshold (1, 1) of _online keys_, or signing keys accessible on-demand by automation for the repository. In fact, they MAY even share the same online key. The metadata for each role MAY expire daily, since it is expected to be updated relatively frequently (e.g., whenever a developer pushes a new bundle / image, or a scheduled job runs to renew the metadata).
```

Comment:
nit: space before _/_ in _bundle/ image_.

Reply:
A difference in style, but I fixed it.